MAUI/Calendar/migration.md

```diff
@@ -289,7 +289,7 @@ this.Content = calendar;
 <tr>
 <td>{{'[DataSource](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_DataSource)'| markdownify }}</td>
```

Comment:
<img width="796" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/46158936/e2200b90-dbb3-4757-a8e6-96b02ebba9f1">


?

Reply:
For Custom labels support, suggested the custom header.